### DISCUSSION TITLE
Any way to access archived tasks?

### DISCUSSION THREAD

**Discussion by archlinux-fan** [Leantime/leantime#549] [Q&A]
I have archived a few tasks today. Can someone explain to me how to access them if I need to re-open them?

**Comment by marcelfolaron** [comment]
You can use the table view. Click "filters" on the top right hand corner and select status: Archived

**Reply by archlinux-fan** [reply]
Oh, I see it now. I guess I was looking in Kanban view only, where there is no status option.
Thank you for your reply.
